rllib/utils/test_utils.py

```diff
@@ -404,8 +404,9 @@ def _test(what, method_to_test, obs_space, full_fetch, explore, timestep,
         if what is trainer:
             # Get the obs-space from Workers.env (not Policy) due to possible
             # pre-processor up front.
-            worker_set = getattr(trainer, "workers",
-                                 getattr(trainer, "_workers", None))
+            worker_set = getattr(trainer, "workers")
```

Comment:
For algos like ES and ARS, that still use `self._workers` instead of `self.workers`.

Reply:
add a TODO to migrate ES and ARS, so we can get rid of this weird if?